swap_meet/vendor.py

```diff
@@ -1,2 +1,90 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory= None):
+        if inventory is None:
+            self.inventory = []
+        else: 
+            self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False 
+    
+    def get_by_id(self,item_id):
```

Comment:
Nit: leave a space after commas
```suggestion
    def get_by_id(self, item_id):
```